stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -82,6 +82,10 @@ public enum _ExecutionMode : Equatable {
 // TODO(hongm): Revisit the longer-term design.
 @_frozen
 public enum _RuntimeConfig {
+  /// When false, tensorflow runtime will be initialized before running any
+  /// tensor program in this process.
+  static public var tensorFlowRuntimeInitialized = false
```

Comment:
From your implementation and description, I can see that this just a flag being used internally within `CompilerRuntime.swift`. So why does this need to be public? `fileprivate` would be most appropriate.

Reply:
The other use-site is in google internal CL 201376415.

I know this kind of PR/CL splitting is prone to confusion. Filed https://bugs.swift.org/browse/SR-8049 to address it in the case of remote session test support.

Reply:
SGTM. Thanks for clarifying!